elements/bulbs-liveblog/bulbs-liveblog-responses.js

```diff
@@ -143,8 +143,8 @@ class BulbsLiveblogResponses extends BulbsHTMLElement {
       this.append(this.responseStaging.firstElementChild);
     }
     window.picturefill();
-    if (twttr) {
-      twttr.widgets.load();
+    if (twttr) {              // eslint-disable-line no-undef
```

Comment:
We should have a readme here with how to use this thing: https://github.com/theonion/bulbs-elements/tree/master/elements/bulbs-liveblog

But we were on a very tight timeline. I don't think this is the PR to add this readme, could make a ticket for it in the backlog.